### PR TITLE
(feat.): add username

### PR DESCRIPTION
**Pull request by AnnaGasparyan1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # tfnt lessons
 
 Tigran Mkrtchyan
+
```
Comment (vre2h) [inline README.md:4]: Why?

**Comment by AnnaGasparyan1** [comment]
Because :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+Anna Gasparyan
```
Comment (vre2h) [inline README.md:5]: Why in english?

**Review by vre2h — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AnnaGasparyan1) [reply README.md:5]: Because :)

**Pull request closed without merging**